src/txn/txn_recover.c

```diff
@@ -959,6 +975,14 @@ __wt_txn_recover(WT_SESSION_IMPL *session, const char *cfg[])
 done:
     /* Close cached cursors, rollback-to-stable asserts exclusive access. */
     WT_ERR(__recovery_close_cursors(&r));
+#ifndef WT_STANDALONE_BUILD
```

Comment:
We should probably add a comment here otherwise @AlexeyAnisimovAU's thought is going to keep coming up to me and others who read this in the future (i.e. that this should be checking for any version <= 10.0.0). Something like:
"There is a known problem with upgrading from release 10.0.0 specifically. There are now fixes that can properly upgrade from 10.0.0 without hitting the problem but only from a clean shutdown of 10.0.0. Earlier releases are not affected by the upgrade issue."

Reply:
Thanks, added the provided comment.